### PR TITLE
rename project to code scanning

### PR DESCRIPTION
**Pull request by TheJuanAndOnly99**
This PR:
- Replaces all occurrences of security-scanning to code-scanning
- Replaces all occurrences of FINOS Security Scanning to FINOS Code Scanning

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -296,15 +296,15 @@ For more info about compliance requirements at FINOS, checkout our [Contribution
 
 ## Roadmap
 1. ~~Add documentation into [community.finos.org](community.finos.org)~~
-2. ~~Publish post on FINOS blog~~ - https://www.finos.org/blog/introducing-finos-security-scanning
+2. ~~Publish post on FINOS blog~~ - https://www.finos.org/blog/introducing-finos-code-scanning
```
Comment (maoo) [inline README.md]: @TheJuanAndOnly99 - please revert this change, the blogpost url should not be changed.

File: README.md [hunk 2]
```diff
@@ -1,14 +1,14 @@
-<img src="https://github.com/finos/finos-landscape/blob/master/hosted_logos/finos-security-scanning.svg" alt="FINOS Security Scanning" width="200"/>
+<img src="https://github.com/finos/finos-landscape/blob/master/hosted_logos/finos-code-scanning.svg" alt="FINOS Security Scanning" width="200"/>
```
Comment (maoo) [inline README.md]: @TheJuanAndOnly99 - are you sure that you changed the name of the logo into the landscape?

**Review by maoo — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (TheJuanAndOnly99) [reply README.md]: reverted

File: README.md (hunk 2 above)
Comment (TheJuanAndOnly99) [reply README.md]: I added a new logo with the name finos-code-scanning.svg. Have not removed the old one yet.

Comment (TheJuanAndOnly99) [reply README.md]: https://github.com/finos/finos-landscape/blob/master/hosted_logos/finos-code-scanning.svg

**Review by maoo — APPROVED** [review]

**Comment by maoo** [comment]
Merging. @josspo - heads up, the Gradle depcheck is failing, see checks above. I've created https://github.com/finos/code-scanning/issues/296 and assigned to you.

**Comment by maoo** [comment]
@TheJuanAndOnly99 - FYI - security-scanning was renamed to code-scanning on Github

**Comment by TheJuanAndOnly99** [comment]
@maoo Thanks!
